vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -71,6 +75,39 @@ def __getitem__(self, arg_name: str):
             return self.type
         return self.figure[arg_name]
 
+    # Interaction methods
+    def _get_figure_interaction_input(self) -> Dict[str, State]:
+        """Required properties when using pre-defined `filter_interaction`."""
+        return {
+            "clickData": State(component_id=self.id, component_property="clickData"),
+            "modelID": State(component_id=self.id, component_property="id"),  # required, to determine triggered model
+        }
```

Comment:
I know that solving "storing model-action info" is the biggest open-question we currently have in the Vizro backend. 

I'm just wondering should we implement `_get_action_input` and `_get_action_output` for every other model?

Reply:
Do you mean that for example the `Dropdown` will tell the code that it's input is the `State(value....)` etc?

Reply:
Exactly, let's continue this "actions_info" conversation in the https://github.com/mckinsey/vizro/pull/289#discussion_r1495590983.